_posts/2021-05-21-pytesting-in-openfaas.md

```diff
@@ -0,0 +1,331 @@
+---
+title: "Testing Python Functions with Pytest"
+description: "Just because you can test in Prod doesn't mean you have to"
+date: 2021-05-15
+image: /images/2021-05-pytesting/conor-samuel-K5BFXOsFp7g-unsplash.jpg
```

Comment:
I crop, then scale width to 1600 then export JPG at 70-75% quality ~ 250-300kb max.

Reply:
Done